### PR TITLE
Add landing message. #1

### PR DESCRIPTION
**Pull request by Artwot**
:)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-	</head>
-	<body>
-		
-	</body>
+  <head>
+    <meta charset="utf-8" />
+    <title>Welcome to Potionfy</title>
+  </head>
+  <body>
+    <h1>Welcome to Pontionfy! Enjoy your stay.</h1>
```
Comment (ge0rg13) [inline index.html:8]: Could you remove the 'Enjoy your stay.' string and make the 'Welcome to potionfy!' red?

Comment (Artwot) [reply index.html:8]: Sure, can do!

**Review by ge0rg13 — CHANGES_REQUESTED** [review]
Please, change the code as per my comments on the review.

**Review by ge0rg13 — APPROVED** [review]
Well done!

**Pull request merged**